x-pack/test/security_solution_ftr/services/detections/index.ts

```diff
@@ -51,6 +62,36 @@ export class DetectionsTestService extends FtrService {
     };
   }
 
+  private async ensureEndpointRuleAlertsIndexExists() {
+    const indexMappings = getAlertsIndexMappings().value;
+
+    if (indexMappings.mappings?._meta?.kibana.version) {
+      indexMappings.mappings._meta.kibana.version = kibanaPackageJson.version;
+    }
+
+    try {
+      await this.esClient.indices.create({
+        index: indexMappings.index,
+        body: {
+          settings: indexMappings.settings,
+          mappings: indexMappings.mappings,
+          aliases: indexMappings.aliases,
+        },
+      });
+    } catch (error) {
+      // ignore error that indicate index is already created
+      if (
+        ['resource_already_exists_exception', 'invalid_alias_name_exception'].includes(
+          error?.body?.error?.type
+        )
+      ) {
+        return;
+      }
```

Comment:
You can also use `indices.exists` before calling `indices.create`. Then you won't have to ignore the `resource_already_exists_exception`. 🤔